lib/thetvdb.py

```diff
@@ -428,7 +428,7 @@ def get_kodishow_details(self, title, serie_details=None):
             result = serie_details
         return result
 
-    def get_kodi_unaired_episodes(self, single_episode_per_show=True, include_last_episode=False):
+    def get_kodi_unaired_episodes(self, single_episode_per_show=True, tvshows_ids=[], include_last_episode=False):
```

Comment:
this is a breaking change... This module is also used in other projects.

Please make this additional parameter an optional filter param.
Also it's bad practice to set the default value to an empty array.

Please change the line to this:

def get_kodi_unaired_episodes(self, single_episode_per_show=True, include_last_episode=False, tvshow_ids=None):